### PR TITLE
Make a delete request 

### PR DESCRIPTION
**Pull request by Suuuuha**
related to #35

### CODE REVIEW

File: src/controllers/places/delete_place.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+module.exports = (req, res) => {
+  const reqOptions = {
+    url: `${placesURL}/${req.params.id}`,
+    method: 'DELETE'
+  };
+
+  Request(reqOptions, (error, response) => {
+    if (error) {
+      res.send('huge failure');
```
Comment (m4v15) [inline src/controllers/places/delete_place.js]: sorry, but lets still send a 404 here (like line 15)

**Review by m4v15 — CHANGES_REQUESTED** [review]
Just that error needs to be change - I might hold off and wait to merge this until #34 is merged because you've included the checkOptions file here (by accident I'm guessing)

File: src/controllers/places/delete_place.js (hunk 1 above)
Comment (jwld) [reply src/controllers/places/delete_place.js]: Should be a 500 yo

File: src/text.js [hunk 2]
```diff
@@ -49,6 +50,7 @@ module.exports = {
     placesView: 'إعرض',
     placesEdit: 'حرر',
     placesAdd: 'اضافة مكان',
+    placesDelet: 'ازالة المكان',
```
Comment (jwld) [inline src/text.js]: typo - missing the final e

**Review by jwld — CHANGES_REQUESTED** [review]
Typo needs fixing

**Pull request merged**